core/integration/features/position_tracking/verified-positions-resolution-5-lognormal.feature

```diff
@@ -57,95 +57,42 @@ Feature: Position resolution case 5 lognormal risk model
     And the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "ETH/DEC19"
 
     Then the parties should have the following account balances:
-      | party  | asset | market id | margin   | general      | bond  |
-      | lpprov | USD   | ETH/DEC19 | 13642884 | 999986267116 | 90000 |
+      | party  | asset | market id | margin  | general      | bond  |
+      | lpprov | USD   | ETH/DEC19 | 6821442 | 999993088558 | 90000 |
 
 
-# insurance pool generation - setup orderbook
+    # insurance pool generation - setup orderbook
     When the parties place the following orders with ticks:
       | party            | market id | side | volume | price | resulting trades | type       | tif     | reference       |
       | sellSideProvider | ETH/DEC19 | sell | 290    | 150   | 0                | TYPE_LIMIT | TIF_GTC | sell-provider-1 |
       | buySideProvider  | ETH/DEC19 | buy  | 1      | 140   | 0                | TYPE_LIMIT | TIF_GTC | buy-provider-1  |
 
-# insurance pool generation - trade
+    # insurance pool generation - trade
     When the parties place the following orders with ticks:
       | party            | market id | side | volume | price | resulting trades | type       | tif     | reference |
       | designatedLooser | ETH/DEC19 | buy  | 290    | 150   | 1                | TYPE_LIMIT | TIF_GTC | ref-1     |
 
 
     Then the parties should have the following account balances:
-      | party  | asset | market id | margin | general       | bond |
-      | lpprov | USD   | ETH/DEC19 | 0      | 1000000000000 | 0    |
-      #| party  | asset | market id | margin        | general | bond |
-      #| lpprov | USD   | ETH/DEC19 | 1000000000000 | 0       | 0    |
+      | party  | asset | market id | margin  | general      | bond  |
+      | lpprov | USD   | ETH/DEC19 | 6907938 | 999993018136 | 90000 |
 
     Then the parties should have the following account balances:
-      | party            | asset | market id | margin  | general |
-      | designatedLooser | USD   | ETH/DEC19 | 21600   | 0       |
+      | party            | asset | market id | margin | general |
+      | designatedLooser | USD   | ETH/DEC19 | 0      | 0       |
```

Comment:
Looks like the party got closed out where previously it was still alive - is this intended?

Reply:
LP bond has been reduced, hence the party did not get close-out, and then close-out happens when the book has enough volume